contracts/StakedCelo.sol

```diff
@@ -137,6 +137,21 @@ contract StakedCelo is ERC20Upgradeable, UUPSOwnableUpgradeable, Managed {
         emit UnlockedStCelo(beneficiary, previouslyLocked - _lockedBalances[beneficiary]);
     }
 
+    /**
+     * @notice Registers transfer to manager whenever stCelo is being transfered.
+     **/
+    function _afterTokenTransfer(
+        address from,
+        address to,
+        uint256 amount
+    ) internal override {
+        if (from == address(0) || to == address(0)) {
```

Comment:
I was wondering if there is any additional accounting that needs to happen during minting/burning, but I guess those only happen in deposit/withdraw, and those functions take care of everything, right?

Reply:
Im not aware of any accounting that would be necessary which would not be already done in deposit/withdraw. Could you maybe elaborate a bit ?